### ISSUE TITLE
Time of share files (Date modified) are inconsistent when Drive share (Redirect)

### ISSUE THREAD

**Issue by maxchendt** [FreeRDP/FreeRDP#5317]
**Describe the bug**
the `Date Modified` on local computer (linuxmint 19.1) and remote computer (windows 10 LTSC 2019) are inconsistent. And the time difference are irregular.

example

File name |  Local time | RDP time |  Difference
------------ | ------------- | ------------- | -------------
R-CDS-2019.tex  | 	 10:48:06  |  10:46  |  2 min lag
AP-intro.tex	 |  21:31:57  |  21:26  |  5 min lag

**To Reproduce**

```
ssh -L 65089:10.9.7.5:3389 server.uk.to -l user -N
xfreerdp /u:user /size:1689x1050 /sec:tls /drive:share,/H/dengta/smb /v:localhost:65089
```

**Expected behavior**
time of files should be the same on local and rdp session.

**Screenshots**
![Screenshot at 2019-03-29 16-55-57](https://user-images.githubusercontent.com/9275768/55221662-f5680700-5244-11e9-8b6c-4cf9a4ec2899.png)

** Application details
* Version of FreeRDP
2.0.0-dev

**Comment by akallabeth** [comment]
Could you please retry with our nightly builds?
There have been some fixes regarding drive redirection.

**Comment by maxchendt** [comment]
freerdp-nightly_2.0.0+0_20190329024841.643_1.gbp52bce4_amd64.deb
yeah, it fix the inconsistency. But, a new bug: If I copy a file from RDP to local, the time is change!

The file `7z-cp.cmd` on E:\  dated `2018-02-18 11:58:28`, is changed to `2018-09-09 10:49:56` . Both the date and time are changed!

![Screenshot at 2019-03-29 21-43-33](https://user-images.githubusercontent.com/9275768/55237068-3cb5be00-526c-11e9-8bdb-25ac055c9e1b.png)

**Comment by akallabeth** [comment]
@maxchendt Ok, fixed in #5318

**Comment by maxchendt** [comment]
on https://ci.freerdp.com/job/freerdp-nightly-binaries/ , Last build is \# 647 (Mar 29, 2019 3:49 AM), where can I find the new build?
For "time change in copying a file from RDP to local", is it fixed yet ?
